litmus-portal/frontend/src/views/CreateWorkflow/WorkflowSettings/__tests__/WorkflowSetting.test.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+import React from 'react';
+import '@testing-library/jest-dom';
+import WorkflowSettings from '../index';
+import { fireEvent, screen } from '@testing-library/react';
+import { render } from '../../../../testHelpers/test-util';
+
+/**
+ * Test for the workflow settings page
+ */
+describe('Workflow Settings', () => {
+  it('Renders InputField', () => {
+    const ref = React.createRef();
+    const { queryByTitle } = render(<WorkflowSettings ref={ref} />);
+    const workflowNameInput = queryByTitle('workflowName')!;
+    const inputValue = workflowNameInput.querySelector('input')!;
+    fireEvent.change(inputValue, { target: { value: 'random text' } });
+    expect(inputValue).toHaveProperty('value', 'random text');
+  });
+
+  it('Displays error text if the length exceeds the threshold', () => {
+    const ref = React.createRef();
+    const { queryByTitle } = render(<WorkflowSettings ref={ref} />);
+    const workflowNameInput = queryByTitle('workflowName')!;
+    const inputValue = workflowNameInput.querySelector('input') as HTMLElement;
+    fireEvent.change(inputValue, {
+      target: {
+        value: 'test-test-test-test-test-test-test-test-test-test-test-',
+      },
```

Comment:
Can you please comment the length of this text as well as the maximum valid length so its easy to read if someone comes/goes through this test